src/modules/comics.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+var request = require('request')
+
+module.exports = {
+  names: ['comic'],
+  onMessage: function (bot, user, userID, channelID, message, rawEvent) {
+    var term = message.split(' ').splice(1).join(' ')
+    if (term === 'c&h') {
```

Comment:
Could be useful to match others as well. Help lists explosm but you can't use 'explosm' as a command